lib/senkyoshi/models/wikipage.rb

```diff
@@ -30,5 +38,31 @@ def canvas_conversion(course, resources)
 
       course
     end
+
+    def _extendeddata(extendeddata)
+      Nokogiri::XML(extendeddata).
+        search("LessonPlanComponent").
+        map do |node|
+          visible = true?(node.search("vislableToStudents").attr("value").to_s)
```

Comment:
@kamijean It is blackboard's mistake. Nice huh? lol
@nick-benoit14 I prefer to avoid xpath as much as possible.